contracts/RouterV2.sol

```diff
@@ -254,7 +255,6 @@ contract Router {
         return swapSynthToBaseFM(inputAmount,  synthIN, msg.sender);
     }
     function swapSynthToBaseFM(uint inputAmount, address synthIN, address member) public returns (uint outPut){
-        require(iSYNTHFACTORY(_DAO().SYNTHFACTORY()).isSynth(synthIN) == true, "!synth");
```

Comment:
This check is inside the Pool's burnSynths() function; no need to double-check the same thing (save gas and refactor)